test/cucumber/support/world.js

```diff
@@ -5,6 +5,9 @@
  * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
  */
 const gavel = require('../../../lib/gavel');
+// DO NOT REMOVE!
+// lodash is not used, but somehow accessed from Cucumber feature testing.
+// May be a case of nasty shadow module references.
 const _ = require('lodash');
```

Comment:
how can the line work if we removed lodash from deps?

Reply:
Okay, I'm testing now, and it seems to pass. Sorry for the confusing comment. Will remove.